api/src/services/users/users.js

```diff
@@ -12,14 +13,16 @@ export const createUser = ({ email, firebaseId }) => {
   return db.user.create({ data: { email, firebaseId } })
 }
 
-export const updateUser = (id, data) => {
+export const updateUser = (data) => {
+  requireAuth()
+
   return db.user.update({
     data: {
-      ...data,
+      ...data.input,
```

Comment:
Then this will become...

```suggestion
      ...input,
```